docs/AOT.md

```diff
@@ -468,6 +468,28 @@ option like in this:
   dot -Tpdf graph.dot -o graph.pdf
   ```
 
+- For debugging purposes you can choose to compile the model in instrumentation
+mode with the option `-instrument-debug` such that the model will display during
+run-time the content of all the tensors associated to all the operands of all the
+IR instructions. Additionally, you can choose the format of the dumped information
+with the option `-instrument-debug-format=<format>`:
+  - `console` (Default) All the operands (tensors) are displayed in text format in the
+  console. Large tensors are only partially displayed.
+  - `bin` The operands (tensors) are dumped in binary format in separate files
+  (one tensor per file). Each file will contain the tensor type and tensor data.
+  - `txt` The operands (tensors) are dumped in text format in separate files
+  (one tensor per file). Each file will contain the tensor type and tensor data.
+  - `rawbin` The operands (tensors) are dumped in binary format in separate files
+  (one tensor per file). Each file will contain ONLY the tensor data.
+  - `rawtxt` The operands (tensors) are dumped in text format in separate files
+  (one tensor per file). Each file will contain ONLY the tensor data.
+  The names of the dump files have a simple format `data[idx].bin` or `data[idx].txt`
+  but a separate meta file `debug.info` is dumped at compile-time which makes the
+  association between each binary file and the operand of the IR instruction to which
+  it belongs. All the dump files and the separate meta file `debug.info` are dumped in
+  a folder `debug` relative to the current directory at compile-time (if it does not
+  exist it is created automatically). You can choose a different directory with the
+  option `-instrument-debug-dir=<dir>`.
```

Comment:
BTW, do we have an example of a textual format produced by these options anywhere in this doc?

Reply:
No, therefore I put some simple examples for `txt` and `rawtxt` formats.